lib/tasks/application_forms.rake

```diff
@@ -34,6 +34,7 @@ namespace :application_forms do
         puts "#{application_form.reference}: #{application_form.action_required_by} - #{application_form.status}"
       end
   end
+<<<<<<< HEAD
```

Comment:
does this need to be removed?

Reply:
Yeah, a rebase that went wrong, I've fixed this now!